src/widgets/tabview/lv_tabview.c

```diff
@@ -107,6 +107,30 @@ void lv_tabview_rename_tab(lv_obj_t * obj, uint32_t idx, const char * new_name)
     lv_label_set_text(label, new_name);
 }
 
+#if LV_USE_TRANSLATION
+
+lv_obj_t * lv_tabview_add_translation_tag_tab(lv_obj_t * obj, const char * tag)
+{
+    LV_ASSERT_OBJ(obj, MY_CLASS);
+
+    lv_obj_t * page = lv_tabview_add_tab(obj, NULL);
+    lv_tabview_rename_translation_tag_tab(obj, -1, tag);
+
+    return page;
+}
+
+void lv_tabview_rename_translation_tag_tab(lv_obj_t * obj, uint32_t idx, const char * new_tag)
```

Comment:
Please use this API name to make it compatible with the editor:
```suggestion
lv_obj_t * lv_tabview_set_tab_translation_tag(lv_obj_t * obj, const char * tag)
```

The order should be `lv_<widget_name>_<set/add/get/bind>_<widget_element>_<property>`
`widget_element` is e.g. a tab of the tabview, series of a chart, etc.

Reply:
done